src/net.cpp

```diff
@@ -1874,7 +1885,7 @@ void CConnman::DisconnectNodes()
             // Disconnect any connected nodes
```

Comment:
_(suggestion at random place in the code)_

Consider to also adjust the `CConnman::DisconnectNode()` functions.

Reply:
Tempted to leave the PR as is for now as it's already quite large.